pytorch_lightning/callbacks/early_stopping.py

```diff
@@ -40,23 +40,18 @@ class EarlyStopping(Callback):
         patience: number of validation epochs with no improvement
             after which training will be stopped. Default: ``3``.
         verbose: verbosity mode. Default: ``False``.
-        mode: one of {auto, min, max}. In `min` mode,
+        mode: one of {min, max}. In `min` mode,
             training will stop when the quantity
             monitored has stopped decreasing; in `max`
             mode it will stop when the quantity
-            monitored has stopped increasing; in `auto`
-            mode, the direction is automatically inferred
-            from the name of the monitored quantity.
-
-            .. warning::
-               Setting ``mode='auto'`` has been deprecated in v1.1 and will be removed in v1.3.
+            monitored has stopped increasing.
```

Comment:
```suggestion
            monitored has stopped increasing. Default: ``'min'``
```

Reply:
Actually, I'd rather remove the defaults as they appear already in the docs from the function signature.

But in another PR. Feel free to take it.